CMakeLists.txt

```
@@ -79,6 +81,43 @@ if(GIT_FOUND)
     string(STRIP "${REPO_SHA1}" DIVE_VERSION_SHA1)
 endif()
 
+# Release type can be set with cmake flag -DDIVE_RELEASE_TYPE=<dev|canary|release|internal>
+set(DIVE_RELEASE_TYPE "dev" CACHE STRING "")
```

Nit: consider moving the comment into the docstring argument of set. This way we serve both readers of the code and users of the GUI
```suggestion
set(DIVE_RELEASE_TYPE "dev" CACHE STRING "Release type can be set with cmake flag -DDIVE_RELEASE_TYPE=<dev|canary|release|internal>")
```